ltp-production.yaml

```diff
@@ -1758,3 +1758,15 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5953
     active: true
     intermittent: true
+  - environments: *environments_all
+    notes: >
+      io_uring01 and io_uring02 fails on compat mode 64-bit kernel booting on 32-bit userspace
+      io_uring01.c:89: TBROK: mmap((nil),388,3,32769,3,0) failed: ENOMEM (12)
+      io_uring02.c:213: TBROK: mmap((nil),18752,3,32769,5,0) failed: ENOMEM (12)
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls/io_uring01
+    - ltp-syscalls/io_uring02
+    url: https://bugs.linaro.org/show_bug.cgi?id=5976
```

Comment:
reported upstream? URL?

Reply:
Yet to report few failures on LTP mailing list and LKML.

Reply:
Reported upstream.